src/services/LogFileManager/index.ts

```diff
@@ -349,28 +349,28 @@ class LogFileManager {
      *
      * @param queryArgs
      * @param queryArgs.queryString
-     * @param queryArgs.isRegex
-     * @param queryArgs.isCaseSensitive
+     * @param queryArgs.queryIsRegex
+     * @param queryArgs.queryIsCaseSensitive
      * @throws {SyntaxError} if the query regex string is invalid.
      */
-    startQuery ({queryString, isRegex, isCaseSensitive}: QueryArgs): void {
+    startQuery ({queryString, queryIsRegex, queryIsCaseSensitive}: QueryArgs): void {
```

Comment:
i feel these lines can be reverted. see the suggestions in `src/typings/query.ts`